theme/templates/features.html

```diff
@@ -45,9 +46,9 @@ <h2 class="heading">{% trans %}Language agnostic Web{% endtrans %}</h2>
         </div>
         <div class="col-xs-12 col-sm-6 col-md-6 col-sm-pull-6">
           <h2 class="heading">{% trans %}Easily Customizable{% endtrans %}</h2>
-          <p>{% trans %}Every business is different and so should be the solution to its problems. Ease of customization is an essential criterion to companies seeking to adopt an ERP system!” {% endtrans %}</p>
-          <p>{% trans %}Most proprietary ERP systems provide some way of extending and customising them. But you can only customize in ways they’ve anticipated – the core is not your property and is not customisable. {% endtrans %}</p>
-          <p>{% trans %}With Tryton you have the ability to customize and modify it as per your specific needs  and business processes. Gone are the times when your business adapt to the enterprise solution, adopt Tryton open source ERP solution to adapt the solution to your business workflows.{% endtrans %}
+          <p>{% trans %}Every business is different and so should be the solution to its problems. Ease of customization is an essential criterion to companies looking to adopt an ERP system! {% endtrans %}</p>
+          <p>{% trans %}Most proprietary ERP systems provide some way of extending and customising their modules. But customizations are limited to the anticipations of the proprietors– the core is not your property and is not customisable. {% endtrans %}</p>
+          <p>{% trans %}With Tryton, you have the ability to customize and modify your solutions as per your specific needs and business processes. Tryton Open Source ERP gives you the independence to customize processes which suit your business workflows. It provides liberty to customize the solutions which suit your business and not the other way around.{%endtrans %}
```

Comment:
Tryton Open Source ERP gives you the independence to customize processes which suit your business workflows. It provides liberty to customize the solutions which suit your business and not the other way around

The move sentence looks repeated.